include/devicetree.h

```diff
@@ -321,21 +395,23 @@
 /**
  * @brief Get a property's logical length
  *
- * Here, "length" is a number of elements, which may not
- * be a size in bytes.
+ * Here, "length" is a number of elements, which may differ from the
+ * property's size in bytes.
```

Comment:
`size in bytes` here doesn't seem right.

Reply:
What is wrong about it? That looks correct to me. The sentence is saying length **may differ** from the size in bytes, i.e. they are not the same